src/api.ts

```diff
@@ -8,9 +8,15 @@ function wait(delay: number) {
   return new Promise(resolve => setTimeout(resolve, delay));
 }
 
-export async function getPeople(): Promise<Person[]> {
+export function getPeople(): Promise<Person[]> {
   // keep this delay for testing purpose
   return wait(500)
     .then(() => fetch(API_URL))
-    .then(response => response.json());
+    .then(response => {
+      if (!response.ok) {
+        throw new Error();
```

Comment:
When throwing an error, it's a good practice to provide an error message to make debugging easier. Consider using `throw new Error('Failed to fetch people data');` to give more context to the error.